src/structures/game_session.ts

```diff
@@ -1187,6 +1187,22 @@ export default class GameSession {
      * @param inVC - Whether the player is currently in the voice channel
      */
     setPlayerInVC(userID: string, inVC: boolean): void {
+        if (
+            inVC &&
+            !this.scoreboard.getPlayerIDs().includes(userID) &&
+            this.gameType !== GameType.TEAMS
+        ) {
+            this.scoreboard.addPlayer(
+                [GameType.CLASSIC, GameType.COMPETITION].includes(this.gameType)
```

Comment:
would it not making more sense to check `this.gameType == elimination`?

Reply:
You're right, thanks :]